lib/rpc.ts

```diff
@@ -76,3 +120,90 @@ export async function createSenderLink(connection: any, senderOptions: SenderOpt
     sender: sender
   };
 }
+
+export function sendRequest(connection: any, link: RequestResponseLink, request: AmqpMessage): Promise<any> {
```

Comment:
>link: RequestResponseLink [](start = 45, length = 25)

this may have been a suggestion from Jeff so it might be a dumb question but why not use classes/methods for the RPC stuff? you could have a send method on the RequestResponseLink object right?